trp_use_case/__openerp__.py

```diff
@@ -20,25 +20,26 @@
 ##############################################################################
 {
     "name": "Use case development",
-    "version": "1.0r45",
-    "author": "Therp BV",
+    "version": "1.0r46",
```

Comment:
while being at it, you can change this to use OCA's versioning scheme: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers